scripts/issue/45/process_bdigg.py

```diff
@@ -0,0 +1,26 @@
+from brownie import interface
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+SLP_TO_WITHDRAW = 5 # ether
+
+def main():
+    """
+    withdraw bdigg slp position and withdraw all to digg
+    """
+    
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_multisig)
+    safe.init_sushi()
+    
+    slp = interface.IUniswapV2Pair(registry.eth.treasury_tokens.slpEthBDigg, owner=safe.account)
+    bdigg = interface.ISettV4h(registry.eth.treasury_tokens.bDIGG, owner=safe.account)
+    digg = interface.ERC20(registry.eth.treasury_tokens.DIGG)
+
+    safe.take_snapshot(tokens=[slp, bdigg, digg])
+    
+    safe.sushi.remove_liquidity(False, slp, SLP_TO_WITHDRAW * 10** slp.decimals())
```

Comment:
perfect, except for here we want to withdraw slp.balanceOf(safe). calling balanceOf here is fine, since it is the first action, and we know that there is no way the balance will change between running simulation onfork (and thus getting hardcoded amount), and executing onchain.